faq/faq.rst

```diff
@@ -34,14 +34,12 @@ As the Citus master node is similar to a standard PostgreSQL server, regular Pos
 How do I ingest the results of a query into a distributed table?
 ----------------------------------------------------------------
 
-Citus currently doesn't support the INSERT / SELECT syntax for copying the results of a query into a distributed table. As a simple workaround, you could use
+Citus now supports the INSERT / SELECT syntax for copying the results of a query
```

Comment:
Link to https://www.postgresql.org/docs/9.6/static/sql-insert.html on INSERT / SELECT